Source/MVC6/Boilerplate.AspNetCore.TagHelpers/Twitter/Cards/TwitterCardPlayer.cs

```diff
@@ -22,58 +22,54 @@ public class TwitterCardPlayer : TwitterCard
+        /// Gets or sets the image to be displayed in place of the player on platforms that don’t
+        /// support iframes or inline players. You should make this image the same dimensions as your
+        /// player. Images with fewer than 68,600 pixels (a 262x262 square image, or a 350x196 16:9
+        /// image) will cause the player card not to render. Image must be less than 1MB in size.
         /// </summary>
-        /// <value>
-        /// The image to be displayed in place of the player on platforms that don’t support iframes or inline players.
-        /// </value>
         [HtmlAttributeName(ImageAttributeName)]
         public TwitterImage Image { get; set; }
 
         /// <summary>
-        /// Gets or sets the video player. If the iframe is wider than 435px, the iframe player will be resized to fit a
-        /// max width of 435px, maintaining the original aspect ratio.
+        /// Gets or sets the video player. If the iframe is wider than 435px, the iframe player will
+        /// be resized to fit a max width of 435px, maintaining the original aspect ratio.
         /// </summary>
-        /// <value>The video player.</value>
         [HtmlAttributeName(PlayerAttributeName)]
         public TwitterPlayer Player { get; set; }
 
         /// <summary>
-        /// Gets or sets the title of the summary. Title should be concise and will be truncated at 70 characters. If you
-        /// are using Facebook's Open Graph og:title, do not use this unless you want a different title.
+        /// Gets or sets the title of the summary. Title should be concise and will be truncated at
+        /// 70 characters. If you are using Facebook's Open Graph og:title, do not use this unless
+        /// you want a different title.
         /// </summary>
-        /// <value>The title of the summary. Title should be concise and will be truncated at 70 characters.</value>
         [HtmlAttributeName(TitleAttributeName)]
         public string Title { get; set; }
 
         /// <summary>
         /// Gets the type of the Twitter card.
         /// </summary>
-        public override TwitterCardType Type => TwitterCardType.Player;
+        public override CardType Type => CardType.Player;
 
         /// <summary>
-        /// Appends a HTML-encoded string representing this instance to the <paramref name="stringBuilder"/> containing
-        /// the Twitter card meta tags.
+        /// Appends a HTML-encoded string representing this instance to the
+        /// <paramref name="stringBuilder"/> containing the Twitter card meta tags.
         /// </summary>
         /// <param name="stringBuilder">The string builder.</param>
         public override void ToString(StringBuilder stringBuilder)
         {
             base.ToString(stringBuilder);
 
+            stringBuilder.AppendMetaNameContentIfNotNull("twitter:site:Id", this.SiteId);
             stringBuilder.AppendMetaNameContentIfNotNull("twitter:title", this.Title);
```

Comment:
It's not a required field, but it is an applicable field.  However if we render it with empty tags the card doesn't render/work properly.
Which is actually the issue I encountered that drew me to contribute.  I was using the cards and it wouldn't render the image for some of the large summary cards.
After picking it apart, I think it was an empty tag for the Site username which I confused with the creator username